code/modules/organs/organ_external.dm

```diff
@@ -491,6 +492,17 @@ This function completely restores a damaged organ to perfect condition.
 		if(config.bones_can_break && !(status & ORGAN_ROBOT) && !(status & ORGAN_PLANT) && brute_dam > min_broken_damage * config.organ_health_multiplier)
 			src.fracture()
 
+		if(fractured)
```

Comment:
Why not just check the ORGAN_BROKEN flag?

Reply:
species that don't have bones have weird ORGAN_BROKEN interactions, from what i know, so having a fractured flag checks specifically whether the fracture proc has fired; it's pretty much to make it foolproof